### PR TITLE
obtain device token

### PR DESCRIPTION
**Pull request by simonepsp**
Hello there,

I’ve recently gotten my first Remarkable (RMPP) and decided to tinker a bit with its (unfortunately quite undocumented) cloud.
I was initially writing my own library but then came across this repo…

In my opinion, we should get this project up and running as soon as possible, as it has amazing potential :) Especially for Goosepaper!

I’ll try to help as much as I can. I’ve written a method to fetch folders and files (API v3) as well but it still needs some cleaning and testing. That'll be a PR for another day.

Cheers,
Simon

### CODE REVIEW

**Comment by j6k4m8** [comment]
omg @simonepsp — I am SO excited to hear this!!! 
Hope you're loving your tablet so far; they're very cool, and even cooler with some open-source magic :) 

Taking a look at the code now!

File: remarkapy/api.py [hunk 1]
```diff
@@ -15,16 +15,22 @@
...
 
 
-class RemarkableAPIError(Exception): ...
+class RemarkableAPIError(Exception):
+    ...
 
 
 class URLS:
     # https://github.com/juruen/rmapi/blob/master/config/url.go
+    # NOTE Webapp endpoints could be fetched on every init
```
Comment (j6k4m8) [inline remarkapy/api.py:24]: woah this is a game-changer!

**Review by j6k4m8 — APPROVED** [review]
This is awesome, good to merge?

**Comment by simonepsp** [comment]
> This is awesome, good to merge?

Glad I could help. Yay let's merge it

**Pull request merged**